servers/propus/default.nix

```diff
@@ -0,0 +1,29 @@
+{ config, pkgs, inputs, lib, ... }: {
+  imports = [
+    inputs.serokell-nix.nixosModules.ec2
+  ];
+
+  networking.hostName = "propus";
+
+  services.jitsi-meet = {
+    enable = true;
+    hostName = "meet.serokell.net";
```

Comment:
I think this should be `meet.serokell.team`? we have a new plan for how we're going to use domains and a good time to start using it would be in this PR so it's not 1 more thing to go back and fix later, I guess we need to formalize the new spec on Notion first